common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -67,6 +67,7 @@ enum eStructsConsts {
     CHANNEL_SCAN_LIST_LENGTH             = 8,
     HT_MCS_SET_SIZE                      = 16,
     VHT_MCS_SET_SIZE                     = 32,
+    DEV_MAX_RADIOS                       = 3,
```

Comment:
Why do we need this? This should be part of the platform, so the correct location for this is not hard coded define but in UCI (nitpick, we can fix it later).

Reply:
actually now i see that we already have a define for this BML_NODE_MAX_RADIOS ....
i'll use it instead